node/slack-command-bot/src/main.js

```diff
@@ -0,0 +1,19 @@
+import { throwIfMissing,throwIfRequestNotVerified,throwIfReplayAttack } from './utils.js';
+
+export default async ({ req, res, log, error }) => {
+  throwIfMissing(process.env, ['SLACK_SIGNING_SECRET']);
+
+  throwIfMissing(req.headers,["x-slack-request-timestamp",'x-slack-signature'])
+
+  const timestamp = req.headers['x-slack-request-timestamp'];
+  throwIfReplayAttack(timestamp)
+  throwIfRequestNotVerified(
```

Comment:
Seems like these functions are basically of the same purpose, can we put the replay check inside the throwIfRequestNotVerified